CNAME

```diff
@@ -1 +0,0 @@
-keys.doppler.com
```

Comment:
@Piccirello -- Am I safe to assume that this is an acceptable change? I believe we're fetching directly from the GitHub repo (via Cloudflare Workers) for both keys.staging.doppler.com and keys.doppler.com

Reply:
This is safe to remove for the reason you mentioned 👍